misc/python/materialize/checks/backup_actions.py

```diff
@@ -0,0 +1,68 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from typing import TYPE_CHECKING
+
+from materialize.checks.actions import Action
+from materialize.checks.executors import Executor
+from materialize.mzcompose.services.minio import MINIO_BLOB_URI
+
+if TYPE_CHECKING:
+    pass
+
+
+class Backup(Action):
+    def execute(self, e: Executor) -> None:
+        c = e.mzcompose_composition()
+        c.exec("mc", "mc", "mb", "persist/crdb-backup")
+        c.exec(
+            "cockroach",
+            "cockroach",
```

Comment:
(I find this a bit special but ok 😉. It was also not changed within this PR.)

Reply:
@nrainer-materialize by "special" do you mean the way SQL is being sent to cockroach?

Reply:
No, I am not too happy about the signature of the invoked method. It apparently often causes the first two arguments to be the same and looks a bit confusing (also see the comment "mc mc mb :D" by def-). I think comprehensibility might be better if `args` (second and following args) were an array but consider that out of scope for this PR.

Reply:
oh ok understood. let me think about this. right now they are args to `docker compose exec` but you want them to be args over `docker compose exec $service_name ...` 

What I wanted to be able to do is `mc_service.create_bucket()` and `cockroach_service.run_sql()` , so I may try to implement something like that in a follow-up refactor.